cc/gtp_player.h

```diff
@@ -47,18 +48,52 @@ class GtpPlayer : public MctsPlayer {
 
     // If true, we will always pass if the opponent passes.
     bool courtesy_pass = false;
+
+    // Number of times to perform tree search for each position in an SGF in
+    // order to evalute a win rate estimation.
+    int num_eval_reads = 4;
```

Comment:
can there be a GTP command (or a flag) for this?

Reply:
added a "ponder winrate N" command